glide/blog.md

```diff
@@ -0,0 +1,147 @@
+TLDR; [Glide](glide-browser.app) is a Firefox-based, [modal](#modes), and [malleable](https://glide-browser.app/config)[^1] web browser for the motivated tinkerer.
+
+##### invisible-heading
+
+[Download](https://glide-browser.app/#download) - [Docs](https://glide-browser.app/) - [Cookbook](https://glide-browser.app/cookbook) - [Source](https://github.com/glide-browser/glide)
+
+---
+
+<br>
+
+I believe browsers should be hackable, just like your editor.
+
+```typescript
+glide.keymaps.set("normal", "gC", async () => {
+  // extract the owner and repo from a url like 'https://github.com/glide-browser/glide'
+  const [owner, repo] = glide.ctx.url.pathname.split("/").slice(1, 3);
+  if (!owner || !repo) throw new Error("current URL is not a github repo");
+
+  // * clone the current github repo to ~/github.com/$owner/$repo
+  // * start kitty with neovim open at the cloned repo
+  const repo_path = glide.path.join(glide.path.home_dir, "github.com", owner, repo);
+  await glide.process.execute("gh", [ "repo", "clone", glide.ctx.url, repo_path]);
+  await glide.process.execute("kitty", ["-d", repo_path, "nvim"], { cwd: repo_path });
+}, { description: "open the GitHub repo in the focused tab in Neovim" });
+```
+
+> Example Glide keymapping to clone the GitHub repo in the current tab and open it in [Kitty](https://sw.kovidgoyal.net/kitty/) + [Neovim](https://neovim.io/).
+
+This snippet captures exactly the kind of functionality I personally want from a browser; extreme customisability so that I can make my daily workflows a joy.
```

Comment:
the wording in this sentence is not great, maybe like

```suggestion
This snippet captures the customizability I expect from my browser so that I can make my daily workflows a joy.
```

i might also expand the connection to other tools (e.g vim)

```
In vim, you could easily make these kinds of customizations tailored to what you need by writing vimscript or installing plugins. In glide, you edit `glide.ts`, which gives a nice typed API. <probably a bit more expansion based on what you want to say/editorialize>
```